package.json

```
@@ -16,16 +16,11 @@
     },
     "private": true,
     "scripts": {
-        "dev-server": "encore dev-server",
-        "dev": "encore dev",
-        "watch": "encore dev --watch",
+        "e2e": "cypress/run.sh",
```

```suggestion
        "e2e": "rm -rf \"cypress/webspace/vendor/contao/manager-bundle\" && composer up --working-dir=\"cypress/webspace\" && symfony server:start --port=8765 --dir=\"cypress/webspace\" --daemon && yarn cypress run --config-file \"cypress/cypress.config.js\" && symfony server:stop --dir=\"cypress/webspace\"",
```

something like this?